assets/js/wcsatt-single-add-to-cart.js

```diff
@@ -31,7 +31,7 @@
 				var $scheme_option = $( this ),
 					scheme_data    = $( this ).find( 'input' ).data( 'custom_data' );
 
-				bundle.satt_schemes.push( { el: $scheme_option, data: scheme_data } );
+				bundle.satt_schemes.push( { el: $scheme_option, data: scheme_data, price_html_o: $scheme_option.find( '.subscription-price' ).html(), details_html: $( '<div>' ).html( $scheme_option.find( '.subscription-details' ) ).html() } );
```

Comment:
> `_o` stands for "original". 

Could the key be changed to `price_html_original` or `price_html_unmodified` then? :)